### PR TITLE
[WIP] Add kind to learning environments 

### PR DESCRIPTION
**Pull request by opowbow**
this refers to git issue #14829 
https://github.com/kubernetes/website/issues/14829

This is still being worked on

### CODE REVIEW

**Comment by opowbow** [comment]
/cc @justaugustus @munnerz @BenTheElder 
for review also....porting over the kind docs to the learning environments pages in kubernetes/website page. 
Let me know if this is the right approach..its still a WIP. thanks!

File: content/en/docs/setup/learning-environment/Kind.md [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+Kubernetes [CRI][CRI] (Container Runtime Interface).
+
+
+## kind Features
+
+Kind supports the following features:
+
+* multi-node (including HA) clusters
+* building Kubernetes release builds from source
+* make / bash / docker, bazel, or installing from apt, in addition to pre-published builds
```
Comment (BenTheElder) [inline content/en/docs/setup/learning-environment/Kind.md]: oops we need to update this upstream, `apt` was removed

**Comment by BenTheElder** [comment]
thanks for the PR and the CC! I'm not sure what the right approach is re porting / linking. :sweat_smile:

**Comment by justaugustus** [comment]
I really like this addition!
My concern here would be keeping the data in sync i.e., is it possible to have our site pull in docs from a repo vs making sure the instructions between here and https://github.com/kubernetes-sigs/kind are always in sync?

Do we have a mechanism for that today, @zacharysarah?

**Comment by zacharysarah** [comment]
@opowbow @justaugustus 👋 
> I really like this addition!

I think the content is great! However, I'm not convinced this PR is the way to add it: see [this comment on the original issue](https://github.com/kubernetes/website/issues/14829#issuecomment-506062149).

> Do we have a mechanism for that today, @zacharysarah?

We don't, short of a submodule approach, and adding a solution would probably be non-trivial. To recapitulate, I'd rather add this information to KinD and link to it from k/website than dual-source it or submodule it.

**Comment by Rajakavitha1** [comment]
> @opowbow @justaugustus 
> 
> > I really like this addition!
> 
> I think the content is great! However, I'm not convinced this PR is the way to add it: see [this comment on the original issue](https://github.com/kubernetes/website/issues/14829#issuecomment-506062149).
> 
> > Do we have a mechanism for that today, @zacharysarah?
> 
> We don't, short of a submodule approach, and adding a solution would probably be non-trivial. To recapitulate, I'd rather add this information to KinD and link to it from k/website than dual-source it or submodule it.

Hi @zacharysarah !!!! The content in this PR is has the same outline as the content in https://kubernetes.io/docs/setup/learning-environment/minikube/.  Kubernetes website has a large footprint and users are looking for information to get started with Kubernetes. Right now we only Minikube as the option to get started with Kubernetes. I agree that it might not be a good idea to recapitulate. However, we could have content to get started with Kubernetes on Kubernetes website and add links for additional information to https://github.com/kubernetes-sigs/kind. In short, we could emulate what https://github.com/kubernetes/minikube has to avoid dual-source or submodule KinD.

**Comment by zacharysarah** [comment]
@Rajakavitha1 
> Right now we only have Minikube as the option to get started with Kubernetes.

...And Minikube tutorial source lives in k/website. This PR proposes to add content from a source outside the k/website chain, effectively dual-sourcing it, but provides no method to automate freshness or ensure that content receives adequate review. 

While I trust that the intentions of everyone involved are 💯 the best, this PR effectively proposes an end run around SIG Docs. That's not a model I want to set up or support, even with the best of intentions.

**Comment by opowbow** [comment]
I agree that dual sourcing is not the best approach going forward....is a better way to simply have a KinD page on the learning-environment page ( so users of k/website are informed of this way to set up a cluster) , but the page consists of links back to the actual KinD pages...as  @zacharysarah  suggested above. The page can be sectioned like the minikube page..and introduced ...but content is wholly linked back to KinD docs? ( again as @zacharysarah suggests ). Thanks for all your helpful comments!

**Comment by zacharysarah** [comment]
@opowbow 👋 Please feel free to `/reopen` when you're ready to continue with feedback.

/close

**Pull request closed without merging**